apiary.apib

```diff
@@ -2696,7 +2698,7 @@ The create request is created and added to a queue, the job resource url for sta
 + Attributes
     + data (optional) - Multipart CSV file upload
     + dataString (optional) - CSV as a string in the request body
-    + incremental (optional) - If incremental is set to 0, the target table is truncated before each import. Default is 0.
+    + incremental (optional, boolean) - If `incremental` is a non-zero value, the import adds data to the existing data in the table. If `incremental` is 0, the target table will be truncated before the import.
```

Comment:
To vím že to se tam dává 1/0, ale je to tu tak definovaný i v ostatních případech, např. hned tady u `withoutHeaders`.